doboto/Droplet.py

```diff
@@ -145,32 +147,30 @@ def shutdown(self, droplet_id, with_tag=None):
         Shutdown a droplet, or tagged droplets
         """
         if with_tag is not None:
-            uri = "%s/actions?tag_name=%s" % (self.uri, with_tag)
+            uri = "{}/actions?tag_name={}".format(self.uri, with_tag)
         else:
-            uri = "%s/%s/actions" % (self.uri, droplet_id)
+            uri = "{}/{}/actions".format(self.uri, droplet_id)
         attribs = {"type": "shutdown"}
         return self.make_request(uri, 'POST', attribs=attribs)
 
     def restore(self, droplet_id, image):
         """
         Restore a droplet from an backup image
         """
-        uri = "%s/%s/actions" % (self.uri, droplet_id)
-
-        try:
-            image = int(image)
-        except ValueError:
-            pass
+        uri = "{}/{}/actions".format(self.uri, droplet_id)
 
-        attribs = {"type": "restore", "image": image}
+        if isinstance(image, int):
+            attribs = {"type": "restore", "image": image}
+        else:
+            attribs = {"type": "restore", "image": "{}".format(image)}
```

Comment:
True, thought this pattern is just for consistency with the variable assignment uses in the other methods.